pom.xml

```diff
@@ -77,11 +78,21 @@
                 <artifactId>liquibase-maven-plugin</artifactId>
                 <version>${liquibase-maven-plugin.version}</version>
                 <configuration>
+                    <!-- use "mvn liquibase:update -X" in Execute Maven Goal -->
                     <changeLogFile>src/main/resources/db/changelog/db.changelog-master.yaml</changeLogFile>
```

Comment:
Can we move this to application.yaml? Why is this included in the pom.xml?

Reply:
No prob. I just wanted to make it more understandable. I can remove everything))

Reply:
All comments are removed.